arb-verifier/test/testArbVerifier.ts

```diff
@@ -46,12 +46,14 @@ describe('ArbVerifier', () => {
     const rollupAddress = process.env.ROLLUP_ADDRESS;
     // When testing against Goerli, replace with this address
     // const rollupAddress = '0x45e5cAea8768F42B385A366D3551Ad1e0cbFAb17';
-
+    const chainId = hre.network.config.chainId
+    console.log(hre.network.config)
+    console.log({chainId})
```

Comment:
it would be better to have a tag before the info, if those console logs are not leftover from debugging